pyproject.toml

```diff
@@ -56,6 +56,7 @@ levenshtein = { version = ">=0.24.0", optional = true }  # This is also used for
 
 # Needed for human evaluation
 gradio = { version = ">=4.26.0", optional = true }
+fasttext = "^0.9.3"
```

Comment:
I don't see the need for anyone using ScandEval to install `fasttext`, since it's only used in the external script. You could instead move it down under development dependencies (and re-do `poetry lock --no-update` after).